src/main/java/org/dinosaur/foodbowl/domain/review/presentation/ReviewControllerDocs.java

```diff
@@ -51,4 +53,30 @@ ResponseEntity<Void> create(
             @Size(max = 4, message = "사진의 개수는 최대 4개까지 가능합니다.") List<MultipartFile> imageFiles,
             Member member
     );
+
+    @Operation(summary = "리뷰 삭제", description = "사용자가 작성한 리뷰를 삭제합니다.")
+    @ApiResponses({
+            @ApiResponse(responseCode = "204", description = "리뷰 삭제 성공"),
```

Comment:
```suggestion
            @ApiResponse(
                    responseCode = "204",
                    description = "리뷰 삭제 성공"
            ),
```
개행 주는 것이 다른 코드와 컨벤션도 맞고 가독성도 좋더라구요 😀